Makefile

```diff
@@ -284,6 +291,7 @@ release: clean-release ## Builds and push container images using the latest git
 .PHONY: release-manifests
 release-manifests: $(RELEASE_DIR) ## Builds the manifests to publish with a release
 	$(KUSTOMIZE) build config/default > $(RELEASE_DIR)/cluster-api-components.yaml
+	$(KUSTOMIZE) build config/kubeadm > $(RELEASE_DIR)/cluster-api-and-bootstrap-components.yaml
```

Comment:
```suggestion
	$(KUSTOMIZE) build config/kubeadm > $(RELEASE_DIR)/cluster-api-components.yaml
```

Reply:
Shouldn't we also append both the core components and the kubeadm ones to this file?

Reply:
Are we also missing the kubeadm standalone components?

Reply:
My understanding was that we'd publish a "batteries included" yaml (kubeadm + core) and a core-only yaml. But I do suppose it's probably a bit better practice to break this up a little further so we can publish core-only, kubeadm-only, and batteries included.

Reply:
I also think I've probably made this more confusing than it should be, with the kubeadm kustomization actually being the new "default". let me move some more things around to try to make this clearer